api/src/main/java/org/openmrs/module/eptsreports/reporting/utils/EptsReportConstants.java

```diff
@@ -139,4 +139,9 @@
     BREASTFEEDINGPREGNANT,
     ADULTCHILDREN
   }
+
+  public enum BreastfeedingAndPregnant {
```

Comment:
Looks like there are no usages for this enum, please delete.

Reply:
@ynurmahomed , done